bigbluebutton-html5/imports/ui/components/chat/chat-dropdown/component.jsx

```diff
@@ -128,6 +128,7 @@ class ChatDropdown extends Component {
             hideLabel
             color="primary"
             aria-label={intl.formatMessage(intlMessages.options)}
+            label={intl.formatMessage(intlMessages.options)}
```

Comment:
Probably don't need to duplicate, you could just have one (`label` lets say) and in the `Button` component you add the `aria-label` the same as the label if not provided.